app/models/review.rb

```diff
@@ -30,8 +30,11 @@ def rating_within_limit
     # - Rating cannot be greater than 5
     # - Rating cannot be a negative number
     # - Handle any potential error/crash
-      if rating > 5 || rating < 1 
-        errors.add(:rating, "Rating can't be greater than 5 and smaller than 1")
-      end
+    if rating === nil
```

Comment:
@fahmiaga Di sini seharusnya bisa pakai line `return unless rating` mirip dengan yang kita lakukan di model Product (`product.rb`). 

Alasan kenapa ngga perlu tampilkan pesan "column cannot be null" adalah karena di model Review (`review.rb`) sudah ada line `validates :rating, presence: true`.